python/pyspark/pandas/series.py

```diff
@@ -6239,13 +6239,19 @@ def argsort(self) -> "Series":
             ps.concat([psser, self.loc[self.isnull()].spark.transform(lambda _: SF.lit(-1))]),
         )
 
-    def argmax(self) -> int:
+    def argmax(self, skipna: bool = True) -> int:
```

Comment:
Good point. But I guess it won't be listed in the missing parameters in the documentation (?)

Reply:
![image](https://user-images.githubusercontent.com/1736354/169432658-ab51f8eb-2014-4d6b-b95d-67e78e077029.png)

It would be listed.

```python
>>> from inspect import signature
>>> signature(pd.Series.argmax).parameters
mappingproxy(OrderedDict([('self', <Parameter "self">), ('axis', <Parameter "axis=None">), ('skipna', <Parameter "skipna: 'bool' = True">), ('args', <Parameter "*args">), ('kwargs', <Parameter "**kwargs">)]))
>>> signature(ps.Series.argmax).parameters
mappingproxy(OrderedDict([('self', <Parameter "self">), ('skipna', <Parameter "skipna: bool = True">)]))
```

Reply:
And the upstream pandas's `axis` is also a dummy parameter (this is really special). So we just keep same with pandas doc/parameters is enough.

Reply:
Yeah, maybe we can add that parameter, and document which value in `axis` is not supported then.

Reply:
Thank you @Yikun @HyukjinKwon ! `axis` is added.